codes/extractive/netsum/netsum.py

```diff
@@ -43,18 +47,28 @@ def data_prep(text_path, summary_path, file_info, save_transformed_flag, transfo
             text = rf.read()
             text_sents = sent_tokenize(text)
             all_docs_words = word_tokenizer.tokenize(text)
-            text_word_counter = Counter(all_docs_words)
-            all_docs_bigram = list(nltk.bigrams(all_docs_words))
-            total_num_words = len(all_docs_words)
+            all_docs_words_cleaned = []
+            # add lemmatization here.
```

Comment:
is this stemming?

Reply:
No. These are some helper methods so that I can calculate sumbasic score and sumbasic bigram score.